packages/js/product-editor/src/components/product-section-layout/product-field-section.tsx

```diff
@@ -15,6 +15,7 @@ type ProductFieldSectionProps = {
 	title: string;
 	description: string | JSX.Element;
 	className?: string;
+	children: JSX.Element | JSX.Element[];
```

Comment:
While I don't overly mind this, should we use `React.ReactElement` instead?
Or instead of using `React.FC`, just setting the props using: `React.PropsWithChildren< ProductFieldSectionProps >`

Reply:
Fixed with 0822966434e46bd39dbbcddba65d7b2149c5092a